raster-foundry/raster-foundry
# Add STAC Export Builder in Batch

## Overview

This branch contains the work in progress batch job

### Checklist

- ~Styleguide updated, if necessary~
- ~Swagger specification updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~

## Testing Instructions

- How to test this PR
- Prefer bulleted description
- Start after checking out this branch
- Include any setup required, such as bundling scripts, restarting services, etc.
- Include test case, and expected output

Closes #5057 
Closes #5086
